cli/src/main/java/io/hyperfoil/cli/commands/WrkAbstract.java

```diff
@@ -220,6 +220,7 @@ public CommandResult execute(HyperfoilCommandInvocation invocation) {
                      .protocol(protocol).host(uri.getHost()).port(protocol.portOrDefault(uri.getPort()))
                      .allowHttp2(enableHttp2)
                      .sharedConnections(connections)
+                     .useHttpCache(false)
```

Comment:
We can still have it opt-in adding a specific wrk argument to enable it back, wdyt?
this will help you to check with/without and users as well

Reply:
yeah, in this case, it could be handy - as long as there are no conflicting named wrk2 args

Reply:
Added as `--use-http-cache`, by default it is set to `false`.